jibri/rootfs/defaults/autoscaler-sidecar.config

```diff
@@ -0,0 +1,14 @@
+export PORT={{ .Env.AUTOSCALER_SIDECAR_PORT }}
+export GRACEFUL_SHUTDOWN_SCRIPT="/opt/jitsi/jibri/wait_graceful_shutdown.sh"
+export TERMINATE_SCRIPT="/opt/jitsi/jibri/shutdown.sh"
+export RECONFIGURE_SCRIPT=
+export ENABLE_REPORT_STATS=true
+export POLLING_URL="{{ .Env.AUTOSCALER_URL  }}/sidecar/poll"
+export STATUS_URL="{{ .Env.AUTOSCALER_URL }}/sidecar/status"
+export STATS_RETRIEVE_URL="http://localhost:2222/jibri/api/v1.0/health"
+export STATS_REPORT_URL="{{ .Env.AUTOSCALER_URL }}/sidecar/stats"
+export ASAP_SIGNING_KEY_FILE="{{ .Env.AUTOSCALER_SIDECAR_KEY_FILE }}"
+export ASAP_JWT_KID="{{ .Env.AUTOSCALER_SIDECAR_KEY_ID }}"
+export INSTANCE_TYPE="jibri"
```

Comment:
Should this be configurable?

Reply:
No this is the type of instance we are running for the jibri container.  For other containers we will set it differently.